### ISSUE TITLE
First step - Proof of Concept

### ISSUE THREAD

**Issue by Laumania** [Laumania/XamlImage#3]
As the first thing, we need to do a Proof of Concept, to figure out is this is actually doable or not.

So, first, create "something" that makes something like this possible.

``` C#
<XamlImage Source="Images/MyVectorImage.xaml" />
```

The above code, or something similar don't feel restricted by having to match the EXACT code from above, it only there to show the idea, should result in the MyVectorImage.xaml being rendered /shown on the page where this "XamlImage" control is inserted.
So basically it should just render another .xaml file inside a .xaml file.
